test/test_common/network_utility.cc

```diff
@@ -88,6 +88,13 @@ const std::string getLoopbackAddressUrlString(const Address::IpVersion version)
   return std::string("127.0.0.1");
 }
 
+const std::string getLoopbackAddressString(const Address::IpVersion version) {
+  if (version == Address::IpVersion::v6) {
+    return std::string("::1");
+  }
+  return std::string("127.0.0.1");
+}
+
 const std::string getAnyAddressUrlString(const Address::IpVersion version) {
   if (version == Address::IpVersion::v6) {
     return std::string("[::]");
```

Comment:
Do we still have other code that uses brackets? And I think we have tests that use brackets too? Can we just remove all bracket use? ntop_ip_loopback_address  vs. ip_loopback_address is kind of confusing to me.